docs/docs/40-operator-guide/40-security/40-managing-credentials.md

```diff
@@ -258,37 +258,53 @@ obtain a token that is valid for 12 hours and cached for 10.
 Kargo can be configured to authenticate to Google Artifact Registry repositories
 using Workload Identity Federation.
 
+#### Cluster Configuration
+
 If Kargo locates no `Secret` resources matching a repository URL, and if Kargo
 is deployed within a GKE cluster, it will attempt to use
 [Workload Identity Federation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
-to authenticate, but this relies upon some external setup. Leveraging this
-option eliminates the need to store credentials in a `Secret` resource.
+to authenticate by default. Leveraging this
+option eliminates the need to store credentials in a `Secret` resource. However, this requires [Workload Identity Federation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) to be enabled on the cluster. You can [enable this when the cluster is created](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_on_cluster) or you can [enable it on an existing cluster that doesn't already have it](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable-existing-cluster). 
 
-First, follow
-[these directions](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_on_cluster)
-to provision a new GKE cluster with Workload Identity Federation enabled or
-[these directions](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable-existing-cluster)
-to enable Workload Identity Federation on an existing GKE cluster.
+:::note
+If you use [GKE Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview), to create the cluster, WIF will be enabled automatically.
```

Comment:
First use of "WIF," although it hasn't been defined. First mention of Workload Identity Federation is up on (new) line 267. Let's introduce the acronym there.